editor/src/components/inspector/common/css-utils.ts

```diff
@@ -3242,48 +3237,46 @@ export function printBackgroundImage(
       }
     }
   })
-  return jsxAttributeValue(backgroundImageStrings.join(' '))
+  return jsxAttributeValue(backgroundImageStrings.join(' '), emptyComments)
 }
 
 export function printBackgroundSize(value: CSSBackgroundSize): JSXAttributeValue<string> {
   const indexOfLastEnabledLayer = findLastIndex(isLayerEnabled, value)
 
-  return jsxAttributeValue(
-    value
+  return jsxAttributeValue(value
       .map((bgSize, i) => {
-        const comma = indexOfLastEnabledLayer > i && bgSize.enabled
-        switch (bgSize.size.value.type) {
+      const comma = indexOfLastEnabledLayer > i && bgSize.enabled
+      switch (bgSize.size.value.type) {
           case 'keyword': {
-            return printEnabled(
-              printComma(printCSSKeyword(bgSize.size.value), comma),
-              bgSize.enabled,
-            )
+              return printEnabled(
```

Comment:
e.g. from here onwards it seems the indenting has changed to 4 spaces rather than 2

Reply:
That does look wrong.